Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM python:3.6.5-jessie
+
+env DEBIAN_FRONTEND noninteractive
```

Comment:
Docker linters do not recognize env if it's in lower case most of the time. If you can just change to 
`ENV DEBIAN_FRONTEND="noninteractive"`
it would be more readable

Reply:
> > Hi @gucharbon, thank you for your review. I am working on the fixes you asked. I just have issues making the image work again (I rebased on top of the mainline). I will keep you updated.
> 
> I mentioned two little things that could still be changed, but you did a really nice job with this new Dockerfile !
> 
> But in any case I'm not reviewer or maintainer of this project, so even if I review it you will still need approval of a maintainer :)

Thanks even more for the review. But how can you be assigned as a reviewer of this PR if you are not a maintainer/reviewer of the project?

Reply:
Oh Okay, I missread the Github interface. It just says that you reviewed my PR but you are not assigned.

Reply:
Yep, just Killing some time :)